packages/components/src/InputFile/InputFile.tsx

```diff
@@ -0,0 +1,232 @@
+import React, { useCallback } from "react";
+import classnames from "classnames";
+import { DropzoneOptions, useDropzone } from "react-dropzone";
+import axios from "axios";
+import uuid from "uuid";
+import styles from "./InputFile.css";
+import { Button } from "../Button";
+import { Content } from "../Content";
+import { Typography } from "../Typography";
+
+interface FileUpload {
```

Comment:
I wish there was a way for us to show this interface in the Docz viewer.

Reply:
Also, if my PR with this interface lands first, we should make sure this PR references that one to avoid duplication.

Reply:
Yea, we should look into how to expose code snippets like this in the docs viewer.

Reply:
`<FormatFile>` landed in master, so we should import it here or export this here.